builtin/settingtypes.txt

```diff
@@ -819,6 +819,9 @@ server_url (Server URL) string https://minetest.net
 #    Automatically report to the serverlist.
 server_announce (Announce server) bool false
 
+#    Replace all player names send to tyhe server list with "anon#".
+server_anon_name (Hide player names on the serverlist) bool false
```

Comment:
i'd called it server_announce_anonymize_names to be explicit

Reply:
typo: 'send to tyhe server' -> send to the server'

Reply:
Fixed in 33c9c31c7